pkg/cmd/pulumi/new_test.go

```diff
@@ -711,3 +712,68 @@ func TestSetFail(t *testing.T) {
 		})
 	}
 }
+
+func TestErrorIfNotEmptyDirectory(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		desc  string
+		files []string
+		dirs  []string
+		ok    bool
+	}{
+		{
+			desc: "empty",
+			ok:   true,
+		},
```

Comment:
Super nitpick. Feel free leave it how you have it, just pointing out that this could be written like:

```go
	tests := map[string]struct {
		files []string
 		dirs  []string
 		ok    bool
	}{
		"empty": {
			ok: true,
		},
		"non-empty: {
 			files: []string{"foo"},
 			dirs:  []string{"bar"},
 			ok:    false,
 		},
		// ...
	}

	for name, tt := range tests {
 		tt := tt
 		t.Run(name, func(t *testing.T) {
 			// ...
 		}
	}
```

Reply:
Thanks. I don't feel too strongly about it, but I've generally preferred the slice form over the map form. 
However, I don't have a very clear reason for why yet, so I'm happy to change it if you have a preference for the map.
FWIW all other table tests in the file (and the ones I saw elsewhere in the codebase) use the slice form.

Reply:
The slice form is definitely more consistent with the rest of our code base, I would keep it like that.